PWGHF/DataModel/CandidateSelectionTables.h

```diff
@@ -296,6 +296,102 @@ bool selectionPID(const T1& pidTrackPi, const T2& acceptPIDNotApplicable)
 DECLARE_SOA_TABLE(HfSelB0ToDPi, "AOD", "HFSELB0", //!
                   hf_sel_candidate_b0::IsSelB0ToDPi);
 
+namespace hf_sel_candidate_bs
+{
+DECLARE_SOA_COLUMN(IsSelBsToDsPi, isSelBsToDsPi, int); //!
+
+/// Apply topological cuts as defined in SelectorCuts.h
+/// \param candBs Bs candidate
+/// \param cuts Bs candidate selection per pT bin"
+/// \param binsPt pT bin limits
+/// \return true if candidate passes all selections
+template <typename T1, typename T2, typename T3>
+bool selectionTopol(const T1& candBs, const T2& cuts, const T3& binsPt)
+{
+  auto ptCandBs = candBs.pt();
+  auto ptDs = RecoDecay::pt(candBs.pxProng0(), candBs.pyProng0());
+  auto ptPi = RecoDecay::pt(candBs.pxProng1(), candBs.pyProng1());
+
+  int pTBin = findBin(binsPt, ptCandBs);
+  if (pTBin == -1) {
+    LOGF(info, "Bs topol selection failed at getpTBin");
```

Comment:
I would remove this message since it could be repeated many times if your binning does not start from 0 (hence it is probable to get `-1` from the `findBin` function)
```suggestion
```